packages/legend-query-builder/src/stores/QueryBuilderState.ts

```diff
@@ -161,6 +161,10 @@ export abstract class QueryBuilderState implements CommandRegistrar {
   // we probably want to refactor to an extension mechanism
   TEMPORARY__setupPanelContentRenderer?: (() => React.ReactNode) | undefined;
 
+  TEMPORARY__templateQueryPanelContentRenderer?:
```

Comment:
can we do this via extension please, i dont want another temproary here we need to clean up